snippets/csharp/VS_Snippets_CLR_System/system.string.isnullorempty/cs/isnullorempty1.cs

```diff
@@ -14,7 +14,7 @@ private static bool TestForNullOrEmpty(string s)
    {
       bool result;
       // <Snippet1>
-      result = s == null || s == String.Empty;
+      result = string.IsNullOrEmpty(s);
```

Comment:
I guess this change should be reverted, because it's for this part of the doc:
![image](https://user-images.githubusercontent.com/15279990/57197208-c5f39b00-6f64-11e9-8333-004d7482adb9.png)

https://docs.microsoft.com/en-us/dotnet/api/system.string.isnullorempty?view=netframework-4.8#remarks